src/services/utilServices/SitesService.js

```diff
@@ -47,7 +68,7 @@ class SitesService {
       })
     )
 
-    const sites = await Bluebird.map(paramsArr, async (params) => {
+    const allSites = await Bluebird.map(paramsArr, async (params) => {
```

Comment:
nit: `allSites` reads slightly weird but as i have no good name, i'm ok w/ just leaving this as is ._.